pre_processing/read_slstr_funcs.F90

```diff
@@ -191,8 +191,11 @@ subroutine read_slstr_tirdata(indir, inband, outarr, sx, sy)
 
 end subroutine read_slstr_tirdata
 
-! Read the nadir-view visible grid data from SLSTR. Need to update with oblique view
-! This assumes that VIS data is twice resolution of TIR (should be correct, 0.5km)
+! Read the nadir-view visible grid data from SLSTR. Need to update with
```

Comment:
I'm guessing the odd-row-numbers happens because of the rectification done at L0 to make coastlines look realistic in the rectangle. This makes me concerned if our method for accumulating visible radiance is appropriate, given we're working on data that's got cosmetic fill pixels and georectification in it. We can't reasonably do this at L0, but should we worry about orphaned pixels or rows that have been shifted a lot relative to their neighbours?

Reply:
You are right, it is a bit worrying. I'm pretty sure this is not meant to happen and I don't think it is documented, so who knows what the best thing to do is!?
My guess would be that we're unlikely to be more than 1 out on the VIS pixel grid, which is probably better than the co-registration between the IR and VIS grid in the L1B data anyway.